firmware/Src/synth/Timbre.cpp

```diff
@@ -1960,27 +1963,37 @@ void Timbre::fxAfterBlock() {
                 delayReadPos = modulo2(delayReadPos - 512, delayBufStereoSize);
                 delayOut2 = delayInterpolation(delayReadPos, delayBuffer_, delayBufStereoSizeM1);
                
+                delayWritePosF += sampleRateDivideInv;
+
                 // lp L
                 low3  += f2 * band3;
-                band3 += f2 * ((delayOut1) - low3 - band3);
+                band3 += f2 * (delayOut1 - low3 - band3);
+                low4  += f2 * band4;
+                band4 += f2 * (low3 - low4 - band4);
                 low5  += f2 * band5;
-                band5 += f2 * ((low3) - low5 - band5);
+                band5 += f2 * (low4 - low5 - band5);
+                low6  += f2 * band6;
+                band6 += f2 * (low5 - low6 - band6);
 
                 // lp R
-                low4  += f2 * band4;
-                band4 += f2 * ((delayOut2) - low4 - band4);
-                low6  += f2 * band6;
-                band6 += f2 * ((low4) - low6 - band6);
+                low9  += f2 * band9;
```

Comment:
I must miss something.
How can we have only "+=" on the low* and band* variable ?
We never reset their value ? (except when we instanciate the object).

Reply:
That's the way these lowpass filters works : input is fed into the band* variable then read with the low* var.

Reply:
i reckon i do not fully understand what's happening within these filters equations, but it works !